.changeset/nice-cameras-march.md

```diff
@@ -0,0 +1,9 @@
+---
+"wrangler": patch
+---
+
+fix: make it possible for values in vars and defines to have colons (:)
+
+Prior to this change, passing --define someKey:https://some-value.com would result in an incomplete value being passed to the Worker.
+
+This change correctly handles colons for var and define in `wrangler dev` and `wrangler publish`.
```

Comment:
It would be great to see a test that proved the bug was fixed. E.g. running a `publish` action with a `--define a:b:c` and checking that `a` gets replaced correctly in the built code.